src/components/radio/radio-item/RadioItem.tsx

```diff
@@ -39,31 +41,47 @@ const radioOverrides = ({
   },
   RadioMarkOuter: {
     component: icon && RadioIcon(icon),
-    style: ({ $theme, $checked }: { $theme: CustomThemeType; $checked: boolean }) => ({
-      width: $theme.sizing.scale650,
-      height: $theme.sizing.scale650,
-      backgroundColor: 'transparent',
+    style: ({ $theme, $checked, $disabled }: { $theme: CustomThemeType; $checked: boolean; $disabled: boolean }) => ({
+      width: small ? $theme.sizing.scale500 : $theme.sizing.scale600,
+      height: small ? $theme.sizing.scale500 : $theme.sizing.scale600,
+      backgroundColor: $disabled ? $theme.customColors.light3 : 'transparent',
       ...border({
-        borderColor: $checked ? $theme.colors.primary400 : $theme.colors.contentTertiary,
+        borderColor: $disabled
+          ? $theme.customColors.dark4
```

Comment:
Could you turn this into a separate function? I put in the linter rule about nested ternary expressions because I think it's not very readable